pkg/cluster/nodeutils/util_test.go

```diff
@@ -77,7 +77,7 @@ func TestParseSnapshotter(t *testing.T) {
 		max_container_log_line_size = 16384
 		netns_mounts_under_state_dir = false
 		restrict_oom_score_adj = false
-		sandbox_image = "registry.k8s.io/pause:3.7"
+		sandbox_image = "registry.k8s.io/pause:3.10"
```

Comment:
this test shouldn't need changing, it's just test data. let's not change it and then it won't match a search and replace for the tag next time.

Reply:
Yes, that's right, I've revert it, thank you